docs/ml-features.md

```diff
@@ -729,6 +729,28 @@ for more details on the API.
 </div>
 </div>
 
+## Interaction
+
+`Implements` is a `Transformer` which implements interaction transform.
```

Comment:
"Interaction" is a Transformer ...

I think this examples is valuable but it actually says nothing about what the transformer does. I know it outputs the various interaction features but we should define and give an example here and/or in the comments of the examples.